frontend/src/views/ClientView.vue

```diff
@@ -1,57 +1,137 @@
 <script setup lang="ts">
 import Button from "primevue/button";
+import Calendar from "primevue/calendar";
 import Card from "primevue/card";
+import Dropdown from "primevue/dropdown";
+import FloatLabel from "primevue/floatlabel";
+import InputNumber from "primevue/inputnumber";
 import InputText from "primevue/inputtext";
+import Textarea from "primevue/textarea";
 import { ref } from "vue";
 
-const clientsList = ref([
-  { id: 1, name: "Artem Kovshov" },
-  { id: 2, name: "Sergej Herbert" },
-  { id: 3, name: "Artem Kovshov" },
-  { id: 4, name: "Daria Eliseeva" },
-  { id: 2, name: "Sergej Herbert" },
-  { id: 3, name: "Artem Kovshov" },
-  { id: 4, name: "Daria Eliseeva" },
-]);
+const selectedGender = ref();
+
+let userInfo = ref({
+  name: null,
+  surname: null,
+  street: null,
+  plz: null,
+  city: null,
+  landline: null,
+  mobil: null,
+  email: null,
+  date: null,
+  text: null,
+});
 
-const value = ref(null);
+const genders = ref([
+  { gender: "Herr" },
+  { gender: "Frau" },
+  { gender: "Frau Dr." },
+  { gender: "Herr dr." },
+]);
```

Comment:
Do we need a "ref" for things that do not change?

Also, we already have types available for things like the salutation.

```suggestion
const salutation = Object.values(ClientSalutation);
```